### PR TITLE
Icky 173 - getAllPages

### PR DESCRIPTION
**Pull request by kibo-amolnadagonde**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "nextjs-commerce",
   "version": "1.0.0",
   "scripts": {
-    "dev": "NODE_OPTIONS='--inspect' next dev",
+    "dev": "cross-env NODE_OPTIONS='--inspect' next dev",
```
Comment (okbel) [inline package.json:5]: @goncy !!

Comment (goncy) [reply package.json:5]: Exactly what I'm doing in local, perfect

**Review by lfades — CHANGES_REQUESTED** [review]
Hi @kibo-amolnadagonde 👋  thank you for the PR!

I would like to know the following:
- How can we test it locally? basically steps for getting the required env keys and have a working store with Kibo
- What features of Next.js Commerce are expected to work and which ones won't
- Is the PR ready or are you still working on it?
- Please provide a description in the PR if possible

**Comment by kibo-amolnadagonde** [comment]
removing

**Pull request closed without merging**